### PR TITLE
Added AdaBoostRegressor converter and its unit test

### PR DESCRIPTION
**Pull request by prabhat00155**

### CODE REVIEW

File: tests/test_SklearnAdaBoostConverter.py [hunk 1]
```diff
@@ -41,14 +42,31 @@ def test_ada_boost_classifier_samme(self):
...
-                                      FloatTensorType(X_train.shape))])
+                                      FloatTensorType(X_test.shape))])
         self.assertIsNotNone(model_onnx)
         dump_data_and_model(X_test.astype('float32'), model, model_onnx,
                             basename="SklearnAdaBoostClassifierSAMME",
                             allow_failure="StrictVersion("
                                           "onnxruntime.__version__)"
                                           "<= StrictVersion('0.2.1')")
 
+    @unittest.skip(reason="CumSum op has not yet been implemented in onnx")
```
Comment (wenbingl) [inline tests/test_SklearnAdaBoostConverter.py]: Do you know if there is anyone is working on this onnx Op? If not, we should not enable this kind of converter which can't run with any onnx-runtime.

**Comment by wenbingl** [comment]
@prabhat00155, In general, the converted model should always be able to run with onnx-runtime, otherwise, we should throw an exception during the conversion. This is an agreement among teams.

File: tests/test_SklearnAdaBoostConverter.py (hunk 1 above)
Comment (prabhat00155) [reply tests/test_SklearnAdaBoostConverter.py]: I created this task: https://aiinfra.visualstudio.com/Lotus/_workitems/edit/2740. It is still in unassigned state.

**Comment by prabhat00155** [comment]
In this case, I need an op to calculate Cumulative Sum (https://aiinfra.visualstudio.com/Lotus/_workitems/edit/2740). What do you suggest I do here, wait for onnx to implement the ops, which would take time. Currently the runtime would throw an error.

**Pull request merged**